### ISSUE TITLE
Debug log of SMB shows user credential

### ISSUE THREAD

**Issue by louy2** [hashicorp/vagrant#8665]
### Vagrant version
Vagrant 1.9.5

### Host operating system
Windows 10

### Guest operating system
Alpine Linux 3.6

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "maier/alpine-3.6-x86_64"
  config.vm.synced_folder ".", "/vagrant", type: 'smb'
end
```

### Debug output
I've redacted my credential.
https://gist.github.com/louy2/4a726bfae92367246f94553ce5d53c48

### Expected behavior
Debug log shouldn't include user credential

### Actual behavior
User credential was included. 

### Steps to reproduce
1. `vagrant up --debug`

### References
N/A

**Comment by louy2** [comment]
Please fix this as soon as possible seeing your guideline instructs people to publish their debug log. People may inadvertently publish their credentials.

**Comment by chrisroberts** [comment]
Hi there,

This has been fixed as of Vagrant 2.0.2.

Cheers!
